ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx

```diff
@@ -93,6 +93,9 @@ const Header = ({
           {mediaCollections && (
             <LiveHeaderMedia
               mediaCollection={mediaCollections}
+              eventTrackingData={{
+                componentName: 'live-header-media',
+              }}
```

Comment:
Could we just have this as a `const` in the `LiveHeaderMedia` component itself, rather than passing it as a prop?